features/frameworks/greentea-client/source/greentea_test_env.cpp

```diff
@@ -532,7 +531,7 @@ enum Token {
  * \return Next character from the stream or EOF if stream has ended.
  *
  */
-static int _get_char() {
+extern "C" char greentea_getc() {
```

Comment:
> May be but greentea_serial.cpp looks more like creation of the singleton greentea_serial object. Most of the API ('greentea_') implementation is in greentea_test_env.cpp. I just changed the name.

That is a good point and in some way `greentea_getc` is the counter part of `greentea_send_kv` which is declared in `test_env.h`.